.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+        fail-fast: false
+        max-parallel: 1
+        matrix:
+            python-version: ["3.7", "3.8", "3.9", "3.10"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools
+        python -m pip install rdflib-jsonld
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        python setup.py sdist && pip install ./dist/*
+    - name: Test
+      run: |
+        python -m unittest test/*.py
```

Comment:
```suggestion
        python -m nose test/
```

Reply:
I confirmed I can test even I use unittest instead of nose. So I replace nose with unittest.

Reply:
Okay great, was not sure.